vnext/Microsoft.ReactNative.Cxx/JSValueXaml.h

```diff
@@ -4,7 +4,7 @@
 #pragma once
 #ifndef MICROSOFT_REACTNATIVE_JSVALUEXAML
 #define MICROSOFT_REACTNATIVE_JSVALUEXAML
-#include "../include/CppWinRTIncludes.h"
```

Comment:
>CppWinRTIncludes [](start = 21, length = 16)

It is unfortunate that this dependency was here before. I must have seen it and raise the red flag...

Reply:
do feel free to file an issue to resolve it :)